Source/Dafny/DafnyOptions.cs

```diff
@@ -232,6 +235,16 @@ public enum IncludesModes {
             return true;
           }
 
+        case "compiler": {
+            if (ps.ConfirmArgumentCount(1)) {
+              if (args[ps.i].Length > 0) {
+                CompilerBackends.AddRange(args[ps.i].Split(',').Where(s => s.Length > 0));
```

Comment:
Since `args[ps.i]` is used multiple times within this method, I suggest creating a local variable here.